app/views/study_sessions/show.html.erb

```diff
@@ -12,47 +12,80 @@
   <%= link_to 'End Session',  @studySession,  method: :delete,
   data: {confirm: "Ending the session early will not allow it to count towards your statistics.\nAre you sure?" } %>
   </section>
-
 <body>
+
+  <% if current_user && !current_user.spotify_token.nil?%>
+    <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+    <div class="w3-dropdown-hover">
+      <button class="w3-button w3-black">Choose Playlist</button>
+      <div class="w3-dropdown-content w3-bar-block w3-border">
+        <% @playlists.each do |playlist| %>
+          <a href="<%= playlist.url %>" target="search_iframe"><%= playlist.name %></a> -
```

Comment:
Great work with this - I didn't consider using HTML to achieve this in the way you did!